docs/tutorials/lattice-fermions.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<p>Can you just add a line mentioning that <code>s=None</code> is used to say that they carry no Spin, and if one wanted to do spin-1/2 fermions one could just change <code>s=1/2</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>